python/graphstorm/gconstruct/utils.py

```diff
@@ -219,6 +220,41 @@ def generate_hash():
     random_uuid = uuid.uuid4()
     return str(random_uuid)
 
+
+def update_feat_transformation_conf(conf, feat_dim_dict):
+    """ Update the feature configuration with the feature dimension list
+
+    Parameters:
+    conf: list[dict]
+        Original feature configuration.
+        [{
+            "feature_col":  ["<column name>", ...],
+            "feature_name": "<feature name>",
+            "transform":    {"name": "<operator name>", ...}
+        }]
+    feat_dim_dict: list[tuple] or list[list]
```

Comment:
Is this a list of dict? If it is a dict, please modify the doc string, which seems to be unmatched.